src/EditorFeatures/Core/GoToNextAndPreviousMethodCommandHandler.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Fine.  Moved into CommandHandlers.  I couldn't stand to add another file under `Implementation` though.